crates/jiter-python/src/lib.rs

```diff
@@ -12,7 +12,7 @@ use jiter::{map_json_error, LosslessFloat, PythonParseBuilder, StringCacheMode};
         *,
         allow_inf_nan=true,
         cache_strings=StringCacheMode::All,
-        allow_partial=false,
+        allow_partial=PartialMode::Off,
```

Comment:
Suggest renaming `allow_partial` to `partial_mode`?

Reply:
I wanted to avoid a breaking change, but maybe that's fine here.

Reply:
changed.